sql_generators/serp_events/templates/desktop_query.sql

```diff
@@ -0,0 +1,262 @@
+WITH raw_serp_events AS (
+  SELECT
+    *,
+    mozfun.map.get_key(event.extra, 'impression_id') AS impression_id,
+    event
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_stable.events_v1`,
+    UNNEST(events) AS event
+  WHERE
+    event.category = 'serp'
+    -- allow events related to an impression_id to span 2 submission dates
+    -- we restrict to event sequences started on a single date below
+    AND DATE(submission_timestamp) >= DATE_SUB(@submission_date, INTERVAL 1 DAY)
+    AND DATE_DIFF(DATE(submission_timestamp), @submission_date, DAY) IN (0, 1)
```

Comment:
IIUC `@submission_date` is the most recent date with complete data? In that case, we want this query to cover `@submission_date` and the day before `@submission_date`.

So this should be:
```
AND DATE(submission_timestamp) >= DATE_SUB(@submission_date, INTERVAL 1 DAY)
AND DATE(submission_timestamp) <= @submission_date
```